sda-commons-web-autoconfigure/src/test/java/org/sdase/commons/spring/boot/web/security/handler/ObscuringErrorHandlerTest.java

```diff
@@ -133,6 +133,36 @@ void shouldGetResource() {
         .containsExactly(HttpStatus.OK, "This will not be altered.");
   }
 
+  @Test
+  void shouldGetApiErrorOnNotNullAnnotation() {
+    var headers = new HttpHeaders();
+    headers.setContentType(MediaType.APPLICATION_JSON);
+    var exchange =
+        client.postForEntity(
+            getServerBaseUrl() + "/api/validate",
+            new TestResource().setPostcode("1234"),
+            ApiError.class);
+    assertThat(exchange)
+        .isNotNull()
+        .extracting(ResponseEntity::getStatusCode, r -> r.getBody().getTitle())
```

Comment:
Can we validate the invalidParams field, as well, to check if the response contain the details of the errors?